routes/web.php

```diff
@@ -15,7 +15,5 @@
     return view('welcome');
 });
 
-Route::get('/get_role/{email}', 'UserRoleController@getRole');
-
-/** Allows laravel to be able to serve/host any vue page */
-Route::get('/{any}', 'SpaController@index')->where('any', '.*');
+Route::post('loginVerification', 'LoginController@authenticateUser');
+Route::get('logout','LoginController@logout');
```

Comment:
You need to add back the `get_role` and `/{any}' routes. They are needed for front-end to hit the api and so Laravel can load vue pages properly

Reply:
Not needed for front end, but it is needed for testing api's on our end